packages/config/config/devices/0x0371/zwa024.json

```diff
@@ -0,0 +1,832 @@
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Motion",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "High Temperature",
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "Low Temperature",
+			"maxNodes": 5
+		},
+		"5": {
+			"label": "High Humidity",
+			"maxNodes": 5
+		},
+		"6": {
+			"label": "Low Humidity",
+			"maxNodes": 5
+		},
+		"7": {
+			"label": "High Luminance Level",
+			"maxNodes": 5
+		},
+		"8": {
+			"label": "Low Luminance Level",
+			"maxNodes": 5
+		},
+		"9": {
+			"label": "High UV Level",
+			"maxNodes": 5
+		},
+		"10": {
+			"label": "Low UV Level",
+			"maxNodes": 5
+		},
+		"11": {
+			"label": "Air Temperature (Multilevel)",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "2",
+			"label": "Motion Retrigger Timeout",
+			"valueSize": 2,
+			"unit": "seconds",
+			"minValue": 30,
+			"maxValue": 3600,
+			"defaultValue": 30,
+			"unsigned": true
+		},
+		{
+			"#": "3",
+			"label": "Motion Reset Timeout",
+			"description": "Specifies the interval after which the device will clear its status",
+			"valueSize": 2,
+			"unit": "seconds",
+			"minValue": 30,
+			"maxValue": 3600,
+			"defaultValue": 240,
+			"unsigned": true
+		},
+		{
+			"#": "4",
+			"label": "Motion Sensitivity",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 11,
+			"defaultValue": 11,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "5",
+			"label": "Motion Report Type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 2,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Notification Report",
+					"value": 0
+				},
+				{
+					"label": "Sensor Binary Report",
+					"value": 1
+				},
+				{
+					"label": "Notification and Sensor Binary Report",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "6",
+			"label": "Vibration Sensor",
+			"description": "Enables vibration sensor and specifies the timeout after which the sensor will clear its status; Allowable range: 1-254 minutes",
+			"valueSize": 1,
+			"unit": "minutes",
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Only send vibration triggered notifications",
+					"value": 255
+				}
+			]
+		},
+		{
+			"#": "7",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Vibration Intensity Reports"
+		},
+		{
+			"#": "9",
+			"label": "Wakeup Duration",
+			"valueSize": 1,
+			"unit": "seconds",
+			"minValue": 2,
+			"maxValue": 255,
+			"defaultValue": 15,
+			"unsigned": true
+		},
+		{
+			"#": "10",
+			"label": "Power Status",
+			"unsigned": true,
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": true,
+			"allowManualEntry": false,
```

Comment:
I think this is optional for readonly params. Might be worth removing from your script.